build/templates/rep_caps.rst.mako

```diff
@@ -40,27 +44,43 @@ ${helper.get_rst_header_snippet(name, '-')}
     .. py:attribute:: ${module_name}.Session.${name}[]
 
 % if len(prefix) > 0:
-        If no prefix is added to the items in the parameter, the correct prefix will be added when
-        the driver function call is made.
+        ..
+            If no prefix is added to the items in the parameter, the correct prefix will be added when
+            the driver function call is made.
```

Comment:
I think this is implementation detail that is not useful to the end user.

Reply:
>  It's only visible in the raw source, not in the html documentation.

remove it, it's only causing confusion